Submodels/Parameterizations.py

```
@@ -0,0 +1,775 @@
+Changelog:
+- V1.1: Updated with comments from coderabbitAI. 
+- V1.0: Adapted version of a parameterization using only the bezier coefficients to give an improved fit to the reference data. 
+
+"""
+
+
+
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy import interpolate, optimize
+from typing import Tuple
```

Ruff (0.8.2)

`typing.Tuple` is deprecated, use `tuple` instead

(UP035)